src/Http/Validation/Rules/Username.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Engelsystem\Http\Validation\Rules;
+
+use Respect\Validation\Rules\AbstractRule;
+use Respect\Validation\Validator;
+
+/**
+ * Username validation.
+ * Usernames must have 1-24 chars and only contain alphanumeric chars or "-", "_", "." or "<space>".
+ */
+class Username extends AbstractRule
+{
+    public function validate(mixed $input): bool
+    {
+        $regex = config('username_regex', '/([^\p{L}\p{N}\-_. ]+)/ui');
```

Comment:
I would remove that regex from here as it is confusing (and differs from the config which overwrites it)

Reply:
Du machst mich feddisch… 

> (and differs from the config which overwrites it)

Until now the default config did not contain the regular expression.